### PR TITLE
Fix title not updating after selecting searched probe

### LINKED ISSUE #926: Switching probes doesn't update the head>title
STR:
- Open glam
- Type in search to find probe A, navigate to it
- From the probe detail page, type in probe to find probe B, navigate to it

Probe B's name doesn't get updated in the browser title.

### PR DESCRIPTION
**Pull request by robhudson**
Fixes #926

### CODE REVIEW

File: src/routing/wrappers/Layout.svelte [hunk 1]
```diff
@@ -13,6 +13,15 @@
...
 
   $: isProbeDetailsView = $store.route.section === 'probe';
+
+  const defaultTitle = 'GLAM: Glean Aggregated Metrics Explorer';
+  $: title =
+    $store.route.section === 'probe'
+      ? `${$store.probeName} | GLAM`
+      : defaultTitle;
+  $: {
+    document.title = title;
```
Comment (wlach) [inline src/routing/wrappers/Layout.svelte]: So I'm not a svelte expert, but is this necessary? Shouldn't the `svelte:head` part set document.title?

**Review by wlach — CHANGES_REQUESTED** [review]
Looks reasonable, one question which may be just my ignorance of what Svelte is doing.

File: src/routing/wrappers/Layout.svelte (hunk 1 above)
Comment (robhudson) [reply src/routing/wrappers/Layout.svelte]: It should, yes. Let me test without that and see. I added that before I move the title changes to the script tag after learning that svelte has issues with updating the `<head>` inside an `{#if ...}` clause.

Comment (robhudson) [reply src/routing/wrappers/Layout.svelte]: Updated to remove that. It works just fine without it. Thanks!

**Review by wlach — APPROVED** [review]

**Pull request merged**